spec.md

```diff
@@ -640,9 +640,9 @@ did not include a `"requires":["route_forwarding"]` property.
 
 There are a class of services that provide network storage to applications
 via volume mounts in the application container. A create binding response
-from one of these services MUST include a `volume_mount`.
+from one of these services MUST include `volume_mounts`.
```

Comment:
I'd like such a note to be accompanied by a link to a section in profile.md that specifies what features are (or are not) supported by example platforms. Can we do that in a separate PR?